tests_smoke/smoke_test.py

```diff
@@ -0,0 +1,11 @@
+from test_admin_csv import test_admin_csv  # type: ignore
+from test_admin_one_off import test_admin_one_off  # type: ignore
+from test_api_bulk import test_api_bulk  # type: ignore
+from test_api_one_off import test_api_one_off  # type: ignore
```

Comment:
How come you needed all the `# type: ignore` comments here? Was MyPy complaining?

Reply:
Oh it looks like you can just add a `.` and the type errors are fixed:
`from .test_admin_csv import test_admin_csv`
https://stackoverflow.com/a/4142178